docs/MatrixOne/Reference/Builtin-Functions/Datetime/timediff.md

```diff
@@ -0,0 +1,57 @@
+# **TIMEDIFF()**
+
+## **Description**
+
+`TIMEDIFF()` returns expr1 − expr2 expressed as a time value.
+
+The result returned by `TIMEDIFF()` is limited to the range allowed for TIME values. Alternatively, you can use either of the functions `TIMESTAMPDIFF()` and `UNIX_TIMESTAMP()`, both of which return integers.
+
+## **Syntax**
+
+```
+> TIMEDIFF(expr1,expr2)
+```
+
+## **Arguments**
+
+|  Arguments   | Description  |
+|  ----  | ----  |
+| expr  | Required. expr1 and expr2 are strings which are converted to TIME or DATETIME expressions; these must be of the same type following conversion. Returns NULL if expr1 or expr2 is NULL.|
+
+## **Examples**
+
+```sql
+> select timediff(cast('22:22:22' as time), cast('-11:11:11' as time));
```

Comment:
Why there is no simple and straight-forward example like this one?
mysql> select timediff("22:22:22", "11:00:00");
+------------------------------+
| timediff(22:22:22, 11:00:00) |
+------------------------------+
| 11:22:22.000000              |
+------------------------------+
1 row in set (0.00 sec)